doc/classes/ArrayMesh.xml

```diff
@@ -206,6 +206,8 @@
 			Overrides the [AABB] with one defined by user for use with frustum culling. Especially useful to avoid unexpected culling when using a shader to offset vertices.
 		</member>
 		<member name="shadow_mesh" type="ArrayMesh" setter="set_shadow_mesh" getter="get_shadow_mesh">
+			An optional alternative mesh which is used for rendering shadows. Use for optimization: shadows can be drawn with simpler mesh with only vertex data (without normals, for example).
+			When using [code]Forward+[/code] renderer the shape of the [member shadow_mesh] must match the actual mesh, as the [member shadow_mesh] is used for the depth prepass as well.
```

Comment:
I would hold off on saying this until we have consensus on what the behaviour should be.

Reply:
Ok, removed that last bit for now. We can readd it if the current behaviour will be decided as final.